src/FEMMDeforLinearBaseModule.jl

```diff
@@ -16,8 +16,9 @@ import FinEtools.AssemblyModule: SysvecAssemblerBase, SysmatAssemblerBase, Sysma
 import FinEtools.FEMMBaseModule: FEMMAbstractBase, inspectintegpoints
```

Comment:
The whole thing should be in an FEMM of its own (`FEMMDeforSurfaceABCDamperModule.jl`), fashioned after the Winkler module. Reasons: it is only applicable to surfaces, and it is specific to the deformation model.

Reply:
Sure. I will make a new module for that.